src/server/app/util/services/inactivity-email.server.service.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+let _ = require('lodash'),
+	path = require('path'),
+	q = require('q'),
+	deps = require(path.resolve('./src/server/dependencies.js')),
+	handlebars = require('handlebars'),
+	fs = require('fs'),
+	dbs = deps.dbs,
+
+	configc = deps.config,
+	emailService = deps.emailService,
+	logger = deps.logger,
+	User = dbs.admin.model('User');
+
+
+function buildEmailContent(resource, emailTemplateName, config) {
+	let numOfDays = Math.floor((Date.now() - resource.lastLogin)/config.day);
+	let emailData = {
+		appName: configc.app.instanceName,
+		url: configc.app.url.protocol + '://' + configc.app.url.host + ':' + configc.app.url.port + '/#', // message of the day board
+		name: resource.name,
+		date: /* new Date(resource.lastLogin) */ numOfDays
+	};
+	let emailHTML = fs.readFileSync(`./src/server/app/util/templates/${emailTemplateName}-email.server.view.html`, 'utf-8');
+	let template = handlebars.compile(emailHTML);
+	emailHTML = template(emailData);
+	return emailHTML;
+}
+
+/**
+ * alert users who's accounts have been inactive for 30-89 days. Remove accounts that have been inactive for 90+ days
+ */
+module.exports.run = function(config) {
+
+	// search for users inactive for 30 days
```

Comment:
You are specifying the alert intervals in an array in the config, but you're still pulling out 3 values here. What happens if there are only 2 values in the alertInterval array? What happens if there are 4? I would recommend mapping the alertInterval array to your queries:
`
let alertQueries = config.alertInterval.map((interval) => ({lastLogin: {$lte: new Date(Date.now() - interval), $gt: new Date(Date.now() - interval - config.day)}}));
`

I would also suggest having a separate field with the number of days before deactivating the account entirely, so you don't have to depend on people knowing that the last value in the alertInterval array is the number of days before deactivation (vs warning).

Also, I don't see a need for the config to specify the number of milliseconds in a day. That's something that can be calculated ad-hoc.